docs/js/features/odata/odata-v4-client.mdx

```diff
@@ -26,7 +24,7 @@ For more details on **how to execute requests** using a (pre-)generated OData cl
 
 #### Filter on One-To-Many Navigation Properties
 
-OData V4 introduces the [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any]({useBaseUrl(/api/1.28.1/modules/sap_cloud_sdk_core#any)}).
+OData V4 introduces the [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any](https://sap.github.io/cloud-sdk/api/1.28.1/modules/sap_cloud_sdk_core#any).
```

Comment:
```suggestion
OData V4 introduces [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any](pathname:///api/1.28.1/modules/sap_cloud_sdk_core#any).
```